index.bs

```diff
@@ -11144,23 +11144,25 @@ in which case they are exposed on every object that [=implements=] the interface
 
     1.  Let |steps| be the following series of steps:
         1.  Try running the following steps:
-            1.  Let |O| be <emu-val>null</emu-val>.
+            1.  Let |object| be null.
```

Comment:
Perhaps this change (and below a couple times) should be made separately? Changing the variable name seems somewhat significant.

Reply:
Now depends on #666.